codility_training/lessons.lesson05.PrefixSum.GenomicRangeQuery/sangmandu.py

```diff
@@ -0,0 +1,25 @@
+# you can write to stdout for debugging purposes, e.g.
+# print("this is a debug message")
+
+def solution(S, P, Q):
+    # write your code in Python 3.6
+    pass
+    ret = []
+    for i in range(len(P)):
+        if S.find("A",P[i],Q[i]+1) != -1:
+            ret.append(1)
+        elif S.find("C",P[i],Q[i]+1) != -1:
+            ret.append(2)
+        elif S.find("G",P[i],Q[i]+1) != -1:
+            ret.append(3)
+        else:
+            ret.append(4)
+    return ret

```

Comment:
```suggestion
        temp = S[P[i]:Q[i] + 1]
        if temp.find("A") != -1:
            ret.append(1)
        elif temp.find("C") != -1:
            ret.append(2)
        elif temp.find("G") != -1:
            ret.append(3)
        else:
            ret.append(4)
    return ret
```
이렇게 중복코드 제거할 수 있을 듯 !

Reply:
오호라~ 감사합니당